grapher/builtin.py

```diff
@@ -0,0 +1,86 @@
+import glob # type: ignore (ignore strange redefinition of glob mypy error)
+import os
+import os.path
+import re
+import sys
+
+from .structures import *
+
+module_def_pattern = re.compile(r'static struct PyModuleDef ([A-Za-z0-9_]+)module\s')
+
+class Builtin:
+    def __init__(
+            self,
+            path: str,
+            start: int,
+            end: int,
+            filename: str,
+    ) -> None:
+        self.path = path
+        self.start = start
+        self.end = end
+        self.filename = filename
```

Comment:
I'd suggest to apply `util.normalize()` to make sure file names follow UNIX-style notation